lib/calendar_urls.coffee

```diff
@@ -0,0 +1,83 @@
+ADDRESS_ATTR = ''
+TITLE_ATTR = ''
+
+module.exports = ({ address_attr, title_attr }) ->
```

Comment:
For repenting your snake_case ways, your syntax sins shall be forgiven Cab :pray: